website/config/database.php.example

```diff
@@ -54,10 +54,11 @@ return [
 
         'mysql' => [
             'driver'    => 'mysql',
-            'host'      => env('DB_HOST', 'localhost'),
-            'database'  => env('DB_DATABASE', 'forge'),
-            'username'  => env('DB_USERNAME', 'forge'),
```

Comment:
Same here.

You should probably delete your branch, reset that commit, remove the changes to the example files and commit it and push again. That way your app key isnt in the source history. Message me on skype if you need help.